tensorboard/data/provider.py

```diff
@@ -260,7 +260,7 @@ def read_last_scalars(
         """Read the most recent values from scalar time series.
 
         The most recent scalar value for each tag under each run is retrieved
-        from the latest event (at the latest step).
+        from the latest event (at the latest timestamp).
```

Comment:
What do you think about adding something like this?

`Note that this is different from the sorting used in `read_scalars`, which is by step. This was an accidental misalignment that would need considerable effort to fix across our implementations, so we're leaving it as is for now. In most cases this should not matter, but if the same log dir is used for multiple runs, this might not match the last data point returned by the other method.`

I worry that it might be too many details, but I prefer to make it a bit more clear that this mismatch exist and why. WDYT?

Reply:
Done, thanks so much!